### ISSUE TITLE
Covid-19 Prediction using  ML/DL  techniques

### ISSUE THREAD

**Issue by jeet-Abhi123** [SrijanShovit/HealthLearning#142]
### Health Learning Repository (Proposing new issue)
:red_circle: **Project Title** : Covid-19 prediction using  ML/DL </br>
:red_circle: **Aim** : This dataset was collected  in the initial phases of covid-19 during march and april 2020. The aim of the project is to predict a person is covid-19 +ve or not with minimum possible symtoms like fever, cough, age_above_60, sore_throat etc </br>
:red_circle: **Dataset** : Taken from research paper and provided by Israeli Ministry of Health  https://www.nature.com/articles/s41746-020-00372-6</br>
:red_circle: **Approach** : First will perform EDA on the dataset, and then perform feature engineering on the columns. Will experiment with multiple models like ANN, Gradient Boosting , Decision trees etc. After will perform hyperparameter tuning through KerasTuner and will plot the results graph.

**Kindly assign me the issue under the label of GSSoC'24.**

**Comment by jeet-Abhi123** [comment]
@SrijanShovit  May I know the reason why you have closed this issue?

**Comment by SrijanShovit** [comment]
You really think we need Covid 19 analysis now? Try exploring the repo first, what's going; which issues have been closed.

**Comment by jeet-Abhi123** [comment]
@SrijanShovit  I am not talking about just Covid-19 analysis. 
I am speaking this with a broader vision,  Just look at the dataset. This was collected in the initial phases of the covid like cough, fever, sore throat etc.
**Aim** : If in future some other disease or pandemic will come, our aim is to predict it with minimum possible symptoms.
That is the aim of this project, i.e why it is important for predicting covid-19 with minimum possible symptoms.

**Comment by SrijanShovit** [comment]
Your current comment does not align with this:

🔴 Project Title : Covid-19 prediction using ML/DL

🔴 Aim : This dataset was collected in the initial phases of covid-19 during march and april 2020. The aim of the project is to predict a person is covid-19 +ve or not with minimum possible symtoms like fever, cough, age_above_60, sore_throat etc

Still you can chose some idea on the line of lung/ bronchiole related disease or similar kind of virus. But would need some other dataset generic to these symptoms.
